types/k6/execution.d.ts

```diff
@@ -73,6 +73,8 @@ declare namespace execution {
          * exec.test.abort('this is the reason');
          */
         abort(input?: string): void;
+
+        options: { [key: string]: any };
```

Comment:
it should have the same definition as https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d9f51b9ffac5574ac7afe029b1a095a81c1afc5d/types/k6/options.d.ts#L12 ... not certain if that is possible or how :grimacing:

Reply:
Yeah, I was talking about this with @ppcano and he suggested we just allow any object rather than replicate every possible option...

Reply:
So you can't here replace this with `Options`

```suggestion
        options: Options;
```

? 

Sorry not a typescript person :grimacing: 

cc @oleiade any idea if this is possible?

Reply:
Maybe after we import it :thinking:

Reply:
I believe it should be possible, but the `Options` would need to be declared as an `interface` as I did for Redis: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61666/files#diff-c666546914aff2f3fefe0f9f63a3ec58da0cfe025dcf0ae21a56f6a3c9f065c1R503

Reply:
@mstoykov Could you review again, please? This should be fixed [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61747/commits/e887805ff4993b37a227c5ed8327f2e5ade86eb9). Waiting for your approval to merge.